yellowbrick/text/umap_vis.py

```diff
@@ -45,7 +45,15 @@
 
 
 def umap(
-    X, y=None, ax=None, classes=None, colors=None, colormap=None, alpha=0.7, **kwargs
+    X,
+    y=None,
+    ax=None,
+    classes=None,
+    colors=None,
+    colormap=None,
+    alpha=0.7,
+    show=True,
+    **kwargs
```

Comment:
Thanks for organizing the kwargs in a more readable fashion - I guess this was manual and not black?

Reply:
Can't take any credit here &mdash; it was black!